molecule/system_settings_logging/molecule.yml

```diff
@@ -0,0 +1,67 @@
+---
+scenario:
+    name: system_settings_logging
+    test_sequence:
+        # - dependency not relevant uless we have requirements
```

Comment:
```suggestion
        # - dependency not relevant unless we have requirements
```

Reply:
fixed in https://github.com/puzzle/puzzle.opnsense/pull/61/commits/dd2631325809612384546206a505d8c171a96925

Reply:
and https://github.com/puzzle/puzzle.opnsense/pull/61/commits/63de1c5f3cc8ba2aa4b5c07545a7bd914b8d56a8 :see_no_evil: